core/domain/exp_domain.py

```diff
@@ -280,6 +291,18 @@ class ExplorationChange(change_domain.BaseChange):
             'translation_html'],
         'optional_attribute_names': [],
         'user_id_attribute_names': []
+    }, {
+        'name': CMD_ADD_WRITTEN_TRANSLATION,
+        'required_attribute_names': [
+            'state_name', 'content_id', 'language_code',
+            'needs_update', 'translation_html', 'data_format'],
+        'optional_attribute_names': ['content_html'],
+        'user_id_attribute_names': []
+    }, {
+        'name': CMD_MARK_WRITTEN_TRANSLATION_AS_NEEDING_UPDATE,
```

Comment:
```suggestion
        'name': CMD_MARK_WRITTEN_TRANSLATIONS_AS_NEEDING_UPDATE,
```

As this will mark multiple translations as need_updates

Reply:
Done.